app/components/toolbarSearch/toolbarSearchDirective.js

```diff
@@ -16,7 +16,9 @@
                 link: toolbarSearchLink,
                 restrict: 'EA',
                 templateUrl: 'components/toolbarSearch/toolbarSearchField.tpl.html',
-                scope: {}
+                scope: {
+                    numberKeywords: '='
```

Comment:
do we need two way binding for a number? 
would a simple `@` not be sufficient?